packages/codebytes/src/editor.tsx

```diff
@@ -62,7 +64,7 @@ export const Editor: React.FC<EditorProps> = ({
   const onCopyClick = () => {
     if (!isCodeByteCopied) {
       navigator.clipboard
-        .writeText(text)
+        .writeText(copyFormatter ? copyFormatter(text, language) : text)
```

Comment:
seems to me like the only fix we need is
```suggestion
        .writeText(onCopy ? onCopy?.(text, language) : text)
```
and we can get rid of the rest and just bump the package json in portal app

Reply:
we can rename the prop too if we think it's clearer

Reply:
I initially was going to do it that way, but the return type of `CodebytesChangeHandler` is `void` rather than `string`. I think it's cleaner for `onCopy`, `onEdit`, and `onLanguageChange` to all have the same type. This also allows us to still pass a proper onClick handler for the copy, e.g. if we wanted to add event tracking.

Reply:
definitely down for changing the name of the param, but i don't love keeping onCopy if we're not using it at all, it might be confusing down the line why it's there, and we can always add it back in if we do need it

Reply:
I would agree that if its not being used here to remove it and also understand your reasoning for trying to differentiate it. This might be more of an [argument for Bana's original "on" prop](https://github.com/Codecademy/client-modules/pull/19#issuecomment-1013492076) so all the tracking for each of these actions is packaged together but that can be investigated further in the [tech debt ticket](https://codecademy.atlassian.net/browse/DISC-447).